injective_functions/auction/__init__.py

```diff
@@ -1,6 +1,9 @@
 from decimal import Decimal
 from injective_functions.base import InjectiveBase
 from typing import Dict
+from injective_functions.utils.helpers import (
```

Comment:
I like to order imports from more generic to more specific.. so 
```
from decimal import Decimal
from typing import Dict
from injective_functions.base import InjectiveBase
from injective_functions.utils.helpers import (
```

Reply:
I used a linting tool to automatically manage all imports